docs/source/sptensor.rst

```diff
@@ -1,8 +1,8 @@
 pyttb.sptensor
 ----------------------
 
-.. automodule:: pyttb.sptensor
+.. autoclass:: pyttb.sptensor
```

Comment:
Switching automodule to autoclass drops the auto documentation for the `tt_to_sparse_matrix` and `tt_from_sparse_matrix`. If these are going away/mostly internal then that's ok. Mostly just an fyi

Reply:
These will go away once sptenmat is in place. I tried to align with ktensor, which provided cleaner docs, IMHO.

Reply:
I went back to automodule to include the methods in each of the class files. The class constructor is a bit confusing now, though, at least to me. It states `pyttb.tensor.tensor(...)`, instead of `pyttb.tensor(...)` as it does when stating autoclass.

@ntjohnson1 any thoughts on this?

Reply:
No strong thoughts. If you think the autoclass is cleaner we can probably stick to that for now. This relates to discussions we've had in the past about slightly ambiguous code layout because of our flat structure mapping to the MATLAB version. Probably doesn't need to be resolved for this PR though. We could also pull out some of the other methods from the class files and think about a better place for them so we could use autoclass AND capture the documentation for these methods.

Some more code organization thoughts:
* It's a little confusing we use tensor generically for all multi-dim data but also tensor specifically for dense tensors
* It adds to the ambiguity that we use lowercase class names (to match MATLAB)
* We add convenience imports to flatten the structure, but the docs probably should refer to the actual source location (I am not sure if there is an easy way to convince sphinx otherwise)

Related to the first two, in my repo where I started to play with a rust port (that I haven't touched in months). I was playing with a structure that looks like
```
ttb/
   tensors/
      Dense
      Kruskal
      Sparse
      Tucker
```